### PR TITLE
add more decoding

### PR DESCRIPTION
**Pull request by PocketPi**

### CODE REVIEW

**Review by DaBs — DISMISSED** [review]

File: extension.json [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-      "type": "HighLevelAnalyzer",
-      "entryPoint": "BM64.Hla"
-    }
-  }
-}
+{
+  "name": "BM64 UART",
+  "apiVersion": "1.0.0",
+  "author": "Joakim Langkilde",
+  "version": "1.0.1",
```
Comment (joak0068) [inline extension.json]: Bump version

File: BM64.py [hunk 2]
```diff
@@ -298,81 +592,137 @@ def get_analyzer_frame(self, start_time, end_time, rx_channel):
...
 # High level analyzers must subclass the HighLevelAnalyzer class.
+
+
 class Hla(HighLevelAnalyzer):
+
+    result_types = {
+        'bm64': {
+            'format': '{{data.packet_type}} - {{data.operation}} {{data.data}}'
+        },
+        'BM64 Event': {
```
Comment (joak0068) [inline BM64.py]: keep case style as the others? ```bm64_event```

File: BM64.py [hunk 3]
```diff
@@ -145,34 +145,305 @@
...
-    0x001: "Write Continue Memory Command",
-    0x100: "Lock / Unlock Memory Command",
-    0x111: "Write Memory Command",
-    0x112: "Erase Memory Command",
-}
 
 class Packet(ABC):
     HEADER_FMT = None
-    RESULT_TYPES = None
+    result_type = None
```
Comment (joak0068) [inline BM64.py]: Is this used? remove?

**Review by joak0068 — CHANGES_REQUESTED** [review]
Nice work! 🔥

File: BM64.py (hunk 2 above)
Comment (joak0068) [reply BM64.py]: Also, is this referred to anywhere? It seems like the bm64 events just use the bm64 result type.

**Review by joak0068 — APPROVED** [review]

**Pull request merged**